src/components/frame/ProfileTemplate/ProfileTemplate.css

```diff
@@ -40,7 +40,7 @@
   }
 
   & .mainContainer {
-    border-left: 2px solid var(--grey-blue-0);
+    border-left: 2px solid var(--light-grey);
```

Comment:
For some reason --grey-blue-0 was removed and --light-grey seemed to be the closest choice. Do you think --grey-blue-0 should not have been removed or is there another color that would better replace this?

Reply:
It makes sense, I more meant changing this colour in the context of this PR. But then also that this border can just be hidden on mobile.

Reply:
ProfileTemplate wasn't really part of this PR, i was just fixing this because --grey-blue-0 is missing. Can you confirm how the styles should change and I can create a new issue.

Reply:
@arrenv Just following up and I am a bit confused, I thought the border should only be visible on mobile. Are you saying you want to change it so that the border is removed completely from the mobile view?

Reply:
Yes, when I look at the mobile version, this border sits to the left screen and serves no purpose.

Reply:
@arrenv hiding the left border on mobile now